config/vufind/config.ini

```diff
@@ -98,6 +98,25 @@ asset_pipeline_max_css_import_size = 0
 ; alternate_themes above.  When commented out, no drop-down theme list will display.
 ;selectable_themes = "standard:Standard Theme,mobile:Mobile Theme"
 
+; This array maps each theme to a list of color schemes that it supports.
+; Even if no selectable_color_schemes (below) are enabled for selection in VuFind
+; itself, the user's system (browser or OS) may be set to use dark mode globally.
+; VuFind will only respect a color scheme choice (whether from VuFind or the system)
+; if it's supported by the current theme. If not, the first color theme specified
+; here will be used, or 'light' if there is none specified.
+; If your local theme supports more than light mode, add it to the array below.
```

Comment:
The effect here is that any local themes will disallow dark mode by default, even though it's globally enabled in bootstrap config.  The theme developer would need to actually test its support before adding it to the list below.